<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="regression" duration-ms="8820" started-at="2020-03-26T00:36:09 EDT" finished-at="2020-03-26T00:36:18 EDT">
    <groups>
    </groups>
    <test name="NewUser" duration-ms="8820" started-at="2020-03-26T00:36:09 EDT" finished-at="2020-03-26T00:36:18 EDT">
      <class name="Test.NewUser">
        <test-method status="FAIL" signature="AddingContacts(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Test.NewUser@50b494a6]" name="AddingContacts" duration-ms="8746" started-at="2020-03-26T00:36:09 EDT" finished-at="2020-03-26T00:36:18 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[Said Hamid]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[HoGroups@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[abc123]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[abc123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at Page.SignUpPage.FillTheForm(SignUpPage.java:32)
at Test.NewUser.AddingContacts(NewUser.java:29)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddingContacts -->
      </class> <!-- Test.NewUser -->
    </test> <!-- NewUser -->
  </suite> <!-- regression -->
</testng-results>
